### PR TITLE
Update high traffic images

### PR DESCRIPTION
**Pull request by nwhite222**
Updated images in high traffic DT sections, mostly to reflect preference for dark mode. NR-119134

### CODE REVIEW

File: src/images/scatter-plot-hover.webp
Comment (x8a) [inline src/images/scatter-plot-hover.webp:1]: I think we're not supposed to remove images from the repo. The thing is that the translated files still have reference to the old image, so until the translation script runs to make the imgs updates, the builds will fail. I'd add the image back.

Comment (nwhite222) [reply src/images/scatter-plot-hover.webp:1]: Added it back. Thanks!

**Comment by x8a** [comment]
The new images look great! 💃🏻

I have a question about the images inside the collapsers though. I see they're pretty outdated and in light mode, was the plan to only update the ones outside the collapsers?

**Review by x8a — COMMENTED** [review]
The `distributedtracingFragmentedTrace` screenshot is still in light mode. Do you think we can retake it as well?

**Review by x8a — APPROVED** [review]
> The `distributedtracingFragmentedTrace` screenshot is still in light mode. Do you think we can retake it as well?

Natasha confirmed that she'll come back to this screenshot in a future PR, since she couldn't retake it easily.

**Pull request merged**